egg-purpur--geyser--floodgate.json

```diff
@@ -0,0 +1,64 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-06-08T22:06:12+02:00",
+    "name": "Purpur + Geyser + Floodgate",
+    "author": "support@geysermc.org",
+    "description": "High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.\r\n\r\nThis customized egg will install the GeyserMC and Floodgate plugins as well. This allows Bedrock players to connect to the server without needing a Java account.",
+    "features": [
+        "eula"
+    ],
+    "images": [
+        "ghcr.io/pterodactyl/yolks:java_17",
+        "ghcr.io/pterodactyl/yolks:java_16",
+        "ghcr.io/pterodactyl/yolks:java_11",
+        "ghcr.io/pterodactyl/yolks:java_8",
```

Comment:
There's a comma at the end of the java_8 part that shouldn't be there which results in this failing to import into Pterodactyl.

Reply:
Oh, I didn't notice that